src/components/v2/V2Project/banners/RelaunchFundingCycleBanner/index.tsx

```diff
@@ -0,0 +1,233 @@
+import { Trans } from '@lingui/macro'
+import { Button, Form, Input } from 'antd'
+import { useContext, useState } from 'react'
+
+import Banner from 'components/shared/Banner'
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+import TransactionModal from 'components/shared/TransactionModal'
+import {
+  V2FundAccessConstraint,
+  V2FundingCycleData,
+  V2FundingCycleMetadata,
+} from 'models/v2/fundingCycle'
+import { V2UserContext } from 'contexts/v2/userContext'
+import { BigNumber } from '@ethersproject/bignumber'
+import useProjectTerminals from 'hooks/v2/contractReader/ProjectTerminals'
+import useProjectDistributionLimit from 'hooks/v2/contractReader/ProjectDistributionLimit'
+import useProjectCurrentFundingCycle from 'hooks/v2/contractReader/ProjectCurrentFundingCycle'
+import useProjectSplits from 'hooks/v2/contractReader/ProjectSplits'
+import { useLaunchFundingCyclesTx } from 'hooks/v2/transactor/LaunchFundingCyclesTx'
+
+import {
+  ETH_PAYOUT_SPLIT_GROUP,
+  RESERVED_TOKEN_SPLIT_GROUP,
+} from 'constants/v2/splits'
+import ReconfigurePreview from '../../V2ProjectReconfigureModal/ReconfigurePreview'
+import { ETH_TOKEN_ADDRESS } from 'constants/v2/juiceboxTokens'
+
+export default function RelaunchFundingCycleBanner() {
+  const { projectId } = useContext(V2ProjectContext)
+  const { contracts } = useContext(V2UserContext)
+  const [newDuration, setNewDuration] = useState<BigNumber>(BigNumber.from(0))
+  const [newStart, setNewStart] = useState<string>('0')
+
+  const [modalOpen, setModalOpen] = useState<boolean>(false)
+  const [transactionPending, setTransactionPending] = useState<boolean>(false)
+
+  const { data, loading: deprecatedFundingCycleLoading } =
+    useProjectCurrentFundingCycle({
+      projectId,
+      useDeprecatedContract: true,
```

Comment:
Here we use the flag so we get the "old" data, so that we can pre-propulate our launchFCsFor tx